packages/twenty-server/src/engine/core-modules/billing/types/billing-product-metadata.type.ts

```diff
@@ -0,0 +1,5 @@
+import { BillingAvailablePlanKey } from 'src/engine/core-modules/billing/enums/billing-available-plan-key.enum';
+
+export type BillingProductMetadata = {
+  planKey: BillingAvailablePlanKey;
```

Comment:
don't we also need to identify if a product is the PlanBaseProduct too (the one bearing the base price)

BillingProductMetadata = {
  planKey: BillingAvailablePlanKey;
  isBasePlan?: boolean;
}

what do you think @FelixMalfait @anamarn ?

Reply:
Yes, you are right this will simplify the queries when accessing the billing tables, the structure looks good to me, i'll implement this in my next commit